client/containers/DashboardSubmissions/SubmissionItems.tsx

```diff
@@ -68,39 +85,31 @@ const SubmissionItems = (props: Props) => {
 		},
 	});
 
+	const [localStats, setLocalStats]: [{ [pubId: string]: SubmissionStatus }, any] = useState({});
```

Comment:
Slightly cleaner way to do this:

```tsx
const [localStats, setLocalStats] = useState<{ [pubId: string: SubmissionStatus }>({});
```
this will properly type both items in the `[value, setter]` array.

Reply:
Could we lengthen these names to `localStatuses`? "stats" makes me think it's "statistics"